FileStorage/Advanced.cs

```diff
@@ -37,48 +35,47 @@ public async Task RunFileStorageAdvancedOpsAsync()
             //***** Setup *****//
             Console.WriteLine("Getting reference to the storage account.");
 
-            // Retrieve storage account information from connection string
             // How to create a storage connection string - http://msdn.microsoft.com/en-us/library/azure/ee758697.aspx
-            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(CloudConfigurationManager.GetSetting("StorageConnectionString"));
+            string storageConnectionString = ConfigurationManager.AppSettings.Get("StorageConnectionString");
 
             Console.WriteLine("Instantiating file client.");
             Console.WriteLine(string.Empty);
 
-            // Create a file client for interacting with the file service.
-            CloudFileClient cloudFileClient = storageAccount.CreateCloudFileClient();
+            // Create a share service client for interacting with the file service.
+            ShareServiceClient shareServiceClient = new ShareServiceClient(storageConnectionString);
```

Comment:
Prefer `var` when the type is obvious.

```suggestion
            var shareServiceClient = new ShareServiceClient(storageConnectionString);
```